install/connections.csv

```diff
@@ -25,17 +25,17 @@ AN08,10,"Mount Fleming, Antarctica",,,1999-11-09T00:00:02Z,2000-01-23T00:00:00Z
 AN09,10,"Table Mountain, Antarctica",,,1999-12-07T00:00:02Z,2000-01-21T00:00:00Z
 AN10,10,"Lake Chad, Antarctica",,,1999-12-07T00:00:02Z,2000-01-21T00:00:00Z
 ANWZ,10,Angora Road,,,2008-08-07T00:05:00Z,9999-01-01T00:00:00Z
-APZ,10,The Paps,,,2006-05-07T03:23:54Z,9999-01-01T00:00:00Z
-APZ,20,The Paps,,,2006-05-07T03:23:54Z,2022-04-12T22:07:00Z
+APZ,10,The Paps,,0,2006-05-07T03:23:54Z,9999-01-01T00:00:00Z
+APZ,20,The Paps,,3,2006-05-07T03:23:54Z,2022-04-12T22:07:00Z
 ARAZ,10,Aratiatia Landcorp Farm,,,2007-05-20T23:00:00Z,9999-01-01T00:00:00Z
 ARCZ,10,Arundel,,,2014-05-13T00:05:00Z,9999-01-01T00:00:00Z
 ARHZ,10,Aropaoanui,,,2010-03-11T21:16:00Z,9999-01-01T00:00:00Z
 ARKE,10,Arakihi,Short Period,,1994-07-22T05:17:00Z,1994-12-12T23:16:00Z
 ARPA,10,Arthurs Pass,,,1995-11-11T00:00:01Z,1996-04-07T23:59:59Z
 ARPC,10,Lake Arapuni Road,Short Period,,2001-01-19T23:54:00Z,2001-07-04T07:21:00Z
 ASHV,10,Ash Pit Road,Short Period,,1995-01-11T00:56:00Z,1995-05-28T18:20:00Z
-AUCT,40,Devonport Naval Base,,,2009-03-26T00:00:00Z,9999-01-01T00:00:00Z
-AUCT,41,Devonport Naval Base,,,2009-03-26T00:00:00Z,9999-01-01T00:00:00Z
+AUCT,40,Devonport Naval Base,,0,2009-03-26T00:00:00Z,9999-01-01T00:00:00Z
+AUCT,41,Devonport Naval Base,,1,2009-03-26T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
I wonder if we should add somewhere in the connections readme that "numbers" are generally
short period and broadband: group of 3 [(0,1,2)] 
strong motion: group of 3 (3-4-5)
tsunami: group of 1 (0 for primary, 1 for backup)